src/main/java/com/garbagemule/MobArena/util/EntityPosition.java

```diff
@@ -9,7 +9,6 @@
  * NOTE: I (garbagemule) DID NOT WRITE THIS CLASS (notice the author below)
  * @author creadri
  */
-@SuppressWarnings("serial")
```

Comment:
I had no idea this class existed, wtf...

I think we can safely remove the entire class since it isn't actually used anywhere. I don't think it has ever been in use, because the commit that introduces it (back in 2011) doesn't even use it.

Reply:
OK, do you want me to get rid of it in this PR or do you want to do that in a separate PR?

Reply:
I'll just remove it on the master branch. I think you should try to amend the commit that removed this line, then we won't have issues with rebasing on top of master.